parameterized/parameterized.py

```diff
@@ -40,6 +40,29 @@ def make_method(func, instance, type):
 
 _param = namedtuple("param", "args kwargs")
 
+
+def reapply_patches_if_need(func):
+
+    def dummy_wrapper(orgfunc):
+        @wraps(orgfunc)
+        def dummy_func(*args, **kwargs):
+            return orgfunc(*args, **kwargs)
+        return dummy_func
+
+    if hasattr(func, 'patchings'):
+        func = dummy_wrapper(func)
+        tmp_patchings = func.patchings
+        delattr(func, 'patchings')
+        for _, patch_obj in enumerate(tmp_patchings):
```

Comment:
Is there a reason to use `enumerate` at all? (like, why not just `for patch_obj in tmp_patchings`)?

Reply:
Nothing at all, sorry I will delete it.